locale/en/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -20,7 +20,7 @@ this in further detail later in this topic.
 ## Event Loop Explained
 
 When Node.js starts, it initializes the event loop, processes the
-provided input script (or drops into the [REPL][], which is not covered in
+provided input script (or drops into the [REPL](https://nodejs.org/api/repl.html), which is not covered in
```

Comment:
1. This line exceeds 80 characters limit now.
2. It seems you can just fix the [bottom reference](https://github.com/nodejs/nodejs.org/blame/d990c746d5b9e288cf41b8d8ceaa597f49c9dae6/locale/en/docs/guides/event-loop-timers-and-nexttick.md#L492) if you prefer this URL.